### PR TITLE
Clarify rounding behavior of out-of-bounds src rects for blitFramebuf…

### PR DESCRIPTION
**Pull request by jdashg**
…fer.

Make conformance2/rendering/blitframebuffer-filter-outofbounds.html easier to read.

### CODE REVIEW

File: specs/latest/2.0/index.html [hunk 1]
```diff
@@ -1403,8 +1403,7 @@ <h4>Framebuffer objects</h4>
         SAMPLE_BUFFERS for the draw framebuffer is zero, the samples corresponding to each pixel location in
         the source are converted to a single sample before being written to the destination.<br><br>
 
-        For any source pixel lying outside the read framebuffer, the corresponding destination pixel remains
-        untouched.<br><br>
+        Any distination pixel who's center corresponds to a point outside the source buffer remain untouched.<br><br>
```
Comment (shrekshao) [inline specs/latest/2.0/index.html:1406]: destination typo?

File: sdk/tests/conformance2/rendering/blitframebuffer-filter-outofbounds.html [hunk 2]
```diff
@@ -72,57 +72,51 @@
...
+            { args: [0, 0, 6, 6, 4, 4, 10, 10], changedDstRect: [4, 4, 8, 8], desc: 'only dst region is out-of-bounds, dst region has the same width/height as src region after dst region is clipped to the bounds of draw buffer.'},
+            { args: [-2, -2, 4, 4, 7, 7, 10, 10], changedDstRect: [8, 8, 8, 8], desc: 'both src and dst region are out-of-bounds, dst region has different width/height as src region after dst region is clipped to the bounds of draw buffer.'},
+            { args: [-2, -2, 4, 4, 4, 4, 10, 10], changedDstRect: [6, 6, 8, 8], desc: 'both src and dst region are out-of-bounds, dst region has the same width/height as src region after dst region is clipped to the bounds of draw buffer.'},
+            { args: [-2, -2, 4, 4, 2, 2, 10, 10], changedDstRect: [2 + 2/6*8, 2 + 2/6*8, 8, 8], desc: 'both src and dst region are out-of-bounds. There are some dst pixels (x and y are within [4, 8] , and x or y equals to 4) whose corresponding src pixels are partially inside and partially outside the real sampling area of the src region (the real sampling area is [0, 0, 4, 4]). But the centers of such src pixels are lying outside the real sampling area.'},
```
Comment (kenrussell) [inline sdk/tests/conformance2/rendering/blitframebuffer-filter-outofbounds.html:84]: Could you add more details to the commit description about the changes made to the test's behavior? These are subtle changes and I don't understand them just from reading the code.

**Review by kenrussell — COMMENTED** [review]
Apologies for the long delay reviewing this - I missed it. A few pieces of feedback before approving.

@shrekshao could you please help test the revised test in Chrome and see whether it passes on various platforms?

**Comment by shrekshao** [comment]
This passes on Chrome canary (89.0.4343.0) on Mac/Win/Android(Pixel3XL)

**Comment by kenrussell** [comment]
Thanks @shrekshao for testing and @jdashg for the revised test.

**Pull request merged**